lib/features/game/views/widgets.dart

```diff
@@ -104,10 +114,23 @@ class ResultListItem extends StatelessWidget {
       child: SizedBox(
         height: 50.0,
         child: Row(
-          mainAxisAlignment: MainAxisAlignment.spaceAround,
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
           children: [
-            Expanded(child: Text(playerInfo.name, style: _resultTextStyle)),
-            Expanded(child: Text(playerInfo.score.toString(), style: _resultTextStyle)),
+            Expanded(flex: 1, child: SizedBox()),
+            Expanded(
+                flex: 3,
+                child: Text(
+                  myIndex.toString() + '. ' + playerInfo.name,
```

Comment:
use string interpolation! `$myIndex.${playerInfo.name}`

or better split this Text to 2 first for position second for name and adjust flex accordingly

Reply:
done